Dockerfile.model

```diff
@@ -1,24 +1,24 @@
 FROM php:{PHP_VERSION}-apache
+LABEL 'maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"'
 
-MAINTAINER Thomas Nabord <thomas.nabord@prestashop.com>
-
-ENV PS_DOMAIN <to be defined>
-ENV DB_SERVER <to be defined>
-ENV DB_PORT 3306
-ENV DB_NAME prestashop
-ENV DB_USER root
-ENV DB_PASSWD admin
-ENV ADMIN_MAIL demo@prestashop.com
-ENV ADMIN_PASSWD prestashop_demo
-ENV PS_LANGUAGE en
-ENV PS_COUNTRY gb
-ENV PS_INSTALL_AUTO 0
-ENV PS_DEV_MODE 0
-ENV PS_HOST_MODE 0
-ENV PS_HANDLE_DYNAMIC_DOMAIN 0
-
-ENV PS_FOLDER_ADMIN admin
-ENV PS_FOLDER_INSTALL install
+ENV PS_DOMAIN <to be defined> \
+DB_SERVER <to be defined> \
+DB_PORT 3306 \
+DB_NAME prestashop \
+DB_USER root \
+DB_PASSWD admin \
+ADMIN_MAIL demo@prestashop.com \
+ADMIN_PASSWD prestashop_demo \
+PS_LANGUAGE en  
+PS_COUNTRY gb \
+PS_INSTALL_AUTO 0 \
+PS_ERASE_MYSQL 0 \
+PS_DEV_MODE 0 \
+PS_HOST_MODE 0 \
+PS_HANDLE_DYNAMIC_DOMAIN 0 \
+PS_FOLDER_ADMIN admin \
+PS_FOLDER_INSTALL install \
```

Comment:
I'm okay to merge these ENV statements in one layer, but the format is not the same in that case.

In https://docs.docker.com/engine/reference/builder/#env, it appears we need to add `=`.
I saw they were undefined with the logs:
```
* Reapplying PrestaShop files for enabled volumes ...
/tmp/docker_run.sh: 27: [: -ne: unexpected operator
/tmp/docker_run.sh: 32: [: -ne: unexpected operator
/tmp/docker_run.sh: 37: [: !=: unexpected operator
/tmp/docker_run.sh: 42: [: !=: unexpected operator
/tmp/docker_run.sh: 47: [: =: unexpected operator
/tmp/docker_run.sh: 53: [: =: unexpected operator
```